### PR TITLE
(WIP) Fix scripts for windows

### PR DESCRIPTION
**Pull request by nyrosmith**
DO NOT MERGE!!!

Associated Issue: None

### Summary of Changes
* Added new dep 'npm-run-all'
* Added new script "validate" in package.json
* commented out some code in bin/prettier temporary

### CODE REVIEW

File: bin/prettier.js [hunk 1]
```diff
@@ -16,8 +16,8 @@ const args = [
   "src/*.js",
   "src/*/*.js",
   "src/components/**/*.css",
-  "src/*/!(mochitest)**/*.js",
-  "src/*/!(mochitest)*/**/*.js"
+  // "src/*/!(mochitest)**/*.js",
```
Comment (nyrosmith) [inline bin/prettier.js]: When I comment out this lines prettier script runs on windows. It seems to me that the CLI wants to jump into the globbing which can not be done on Windows

File: package.json [hunk 2]
```diff
@@ -61,7 +61,8 @@
     "precommit": "lint-staged",
     "prepush": "run-p lint test flow",
     "nom": "node bin/nom",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "validate": "npm-run-all --parallel lint flow"
```
Comment (nyrosmith) [inline package.json]: When I get all the scripts working we can document the "validate" script as a starting point after initial cloning & running `yarn install`

File: src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap [hunk 3]
```diff
@@ -400,67 +400,67 @@ Array [
 exports[`Parser.getSymbols finds symbols in an html file 1`] = `
 Array [
   "functions:
- [(8, 2), (10, 3)]   sayHello name
```
Comment (nyrosmith) [inline src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap]: At committing I got the following message:

> warning: LF will be replaced by CRLF in src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap.
> The file will have its original line endings in your working directory.

File: src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap [hunk 4]
```diff
@@ -400,67 +400,67 @@ Array [
 exports[`Parser.getSymbols finds symbols in an html file 1`] = `
 Array [
   "functions:
- [(8, 2), (10, 3)]   sayHello name 
-[(22, 21), (24, 3)]   capitalize name 
-[(36, 3), (39, 3)]   iife  ",
+ [(9, 2), (11, 3)]   sayHello name
```
Comment (jasonLaster) [inline src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap]: lol @jselmani @GarethSharpe we've gotta fix the CLRF [bug][b]

[b]:https://github.com/ryanjduffy/parse-script-tags/issues/3

**Comment by nyrosmith** [comment]
Closing this because I can't get prettier script working on Windows

**Comment by jasonLaster** [comment]
okay... crlf is now in, which should help

**Pull request closed without merging**